data/net/minecraft/util/Mth.mapping

```diff
@@ -138,6 +184,10 @@ CLASS net/minecraft/util/Mth
 		COMMENT Gets the absolute of the difference between two angles in degrees.
 		ARG 0 start
 		ARG 1 end
+	METHOD diffuseLight (FFF)F
+		ARG 0 light1
+		ARG 1 light2
+		ARG 2 light3
```

Comment:
Not sure if this should be mapped as it provides nothing useful about the formula itself.